src/components/auth/register-form.jsx

```
@@ -68,7 +69,7 @@ export default function RegisterForm() {
       setFormError(null)
 
       const { confirmPassword, ...userData } = data
```

ESLint

[error] 'confirmPassword' is assigned a value but never used.

(no-unused-vars)